openlibrary/templates/account/create.html

```diff
@@ -12,12 +12,67 @@ <h1>$_("Sign Up")</h1>
 </div>
 
 <div id="contentBody">
-
 <script type="text/javascript">
 window.q.push( function(){
     \$('#username').keyup(function(){
```

Comment:
This should be `on('keyup'`. This will break with the latest upgrade. Also best to pull this out into a separate commit following the above one.

Longer term you might want to consider using the input event which works with autofill and copy/paste.

Reply:
As discussed in the meeting with @mek and @cdrini we are planning to use `on('blur'` and I think blur should work with autofill and copy/paste as the user will inevitably click out the input box